### DISCUSSION TITLE
Errors while trying to install ADCS server

### DISCUSSION THREAD

**Discussion by avi8892** [Marshall-Hallenbeck/red_team_attack_lab#11] [General]
ADCS server crush and can not boot

![image](https://user-images.githubusercontent.com/31028818/135754440-ac93d52b-102b-4381-a144-9ae4987ad7bf.png)

**Comment by Marshall-Hallenbeck** [comment]
Hey sorry I totally missed this, did you re-try building it? Windows Vagrant hosts are janky and just fail a lot.
